deps/patches/libgit2-ssh.patch

```diff
@@ -0,0 +1,23 @@
+--- CMakeLists.txt	2016-05-26 23:13:48.000000000 -0400
++++ CMakeLists.txt	2016-07-02 23:20:55.842735529 -0400
+@@ -354,14 +354,16 @@
+ 
+ # Optional external dependency: libssh2
+ IF (USE_SSH)
+-	PKG_CHECK_MODULES(LIBSSH2 libssh2)
++	FIND_PACKAGE(Libssh2)
```

Comment:
thanks. if this is yet another msys2 specific issue I'm going to have to migrate appveyor over to cross building from cygwin.